netops_demo/golang/src/github.com/v3io/demos/functions/ingest/ingest.go

```diff
@@ -142,7 +147,8 @@ func ingestMetricSamples(context *nuclio.Context,
 	samples *metricSamples) error {
 	context.Logger.InfoWith("Ingesting",
 		"metricName", metricName,
-		"samples", len(samples.Timestamps))
+		"samples", len(samples.Timestamps),
+		"sample", samples)
```

Comment:
I would revert this - way too much logging

Reply:
Done